Xamarin.Forms.Platform.Android/ViewRenderer.cs

```diff
@@ -223,7 +223,7 @@ protected override bool SetHint()
 			if (_defaultHint == null)
 				_defaultHint = textView.Hint;
 
-			var elemValue = string.Join(". ", (string)Element.GetValue(Accessibility.NameProperty), (string)Element.GetValue(Accessibility.HintProperty));
+            var elemValue = string.Join((String.IsNullOrWhiteSpace((string)(Element.GetValue(Accessibility.NameProperty))) || String.IsNullOrWhiteSpace((string)(Element.GetValue(Accessibility.HintProperty)))) ? "" : ". ", (string)Element.GetValue(Accessibility.NameProperty), (string)Element.GetValue(Accessibility.HintProperty));
```

Comment:
Definitely.. I dont know why it did that when I checked in.  I use tabs as well and my local code has the tabs in there correctly.  For some reason it appears that way on the code after checkin but not in my repository.  Ill reclone and see what comes down.